apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -608,7 +608,7 @@ function handleCloseInLegacyEditors( handleClose ) {
 	const navSidebarCloseSelector = '.wpcom-block-editor-nav-sidebar-toggle-sidebar-button__button';
 	const selector = `.edit-post-header .edit-post-fullscreen-mode-close:not(${ wpcomCloseSelector }):not(${ navSidebarCloseSelector })`;
 
-	const siteEditorSelector = '.edit-site-header .edit-site-fullscreen-mode-close';
+	const siteEditorSelector = '.edit-site-navigation-panel__back-to-dashboard';
```

Comment:
We are trying to phase out these overrides that depend on CSS classnames. The reason is that they can break, and have been broken a couple of times already, during Gutenberg plugin refactors and updates. In my opinion we should use the existing slot/fills instead.

Reply:
#48231 for the slot/fill.  Still has some kinks to work out.